messages/whitespace-found.txt

```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the plugin "eslint-plugin-<%- pluginName %>". It looks like there is whitespace in the plugin name.
```

Comment:
Ah yeah I knew I had it the other way for a reason. You're probably right, but I'll wait for @mysticatea to confirm.